wake/loaders/proxy.ts

```diff
@@ -3,10 +3,13 @@ import { AppContext } from "../mod.ts";
 
 const PATHS_TO_PROXY = [
   ["/checkout", "/checkout"],
+  ["/checkout"],
+  ["/checkout/*"],
```

Comment:
some stores my use checkout path to redirect.

Lets fix extraPathsToProxy to work as intend, so you can set those paths via UI.